notebooks/scripts/Helpers.py

```diff
@@ -98,12 +105,19 @@ def get_hamming_distances(genomes):
     for i in range(len(genomes)):
         # Only compare the current genome, i, with all later genomes.
         # This avoids repeating comparisons or comparing each genome to itself.
+        np_genomes = np.array(list(re.sub("-", "0", genomes[i])))
+
         for j in range(i + 1, len(genomes)):
+            np_genomes_b = np.array(list(re.sub("-", "0", genomes[j])))
+            result = np.where(np_genomes_b == "0")
+            np_genomes[result] = 0
+            a = (np_genomes=="0")
+            num_indel = (a&~np.r_[[False],a[:-1]]).sum()
             # Find all mismatches between these two genomes.
             mismatches = genome_arrays[i] != genome_arrays[j]
 
             # Count the number of mismatches where both genomes have valid bases.
-            hamming_distances.append((mismatches & valid_bases[i] & valid_bases[j]).sum())
+            hamming_distances.append(((mismatches & valid_bases[i] & valid_bases[j]).sum()) + num_indel)
```

Comment:
We want to parameterize the counting of indels so we can enable/disable that counting per pathogen.